script/buildSQLGeom.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+"""
+This module will accept an input text file with (x,y,z) coordinates in a CSV-format list
+and generate a SQL string that will add these coordinates as a PostGIS Polygon
+to sites.geometry within a BETYdb instance.
+
+Command line usage:
+        -h      show help
+        -i      input filename
+        -e      EPSG reference number, default is 3857
+        -s      site_id in BETYdb sites table to associate geometry with
```

Comment:
This is no problem - we'll just need to make sure the user understands that if a sitename is provided, a valid BETY host must also be defined so that the site_id can be queried or created.